content/applications/finance/payment_acquirers/paypal.rst

```diff
@@ -1,109 +1,83 @@
 ======
-Paypal
+PayPal
 ======
 
-`Paypal <https://www.paypal.com/>`_ is available and popular worldwide. It doesn't charge any
-subscription fee, and creating an account is very easy. That's why we recommend it for starters in
-Odoo. It works as a seamless flow where the customer is routed to the Paypal website to register the
-payment.
+`Paypal <https://www.paypal.com/>`_ is an American online payments acquirer available worldwide. As
+it does not charge any subscription fee and creating an account is easy, it is recommended for Odoo
+starters.
 
-Settings in Odoo
-================
-
-.. seealso::
-   - :ref:`payment_acquirers/add_new`
-
-Credentials tab
----------------
-
-Odoo needs your **API Credentials** to connect with your PayPal account, which comprise:
-
-- **Email**: your login email address in Paypal.
-- **PDT Identity Token**: the key used to verify the authenticity of transactions.
-- **Use IPN**: whether you want to use Instant Payment Notification. Already checked, you don't have
-  to change it.
-
-You can copy your credentials from your Paypal account and paste them into the related fields under
-the **Credentials** tab.
+Settings in PayPal
+==================
 
+To access your PayPal account settings, log into PayPal, open the :guilabel:`Account Settings`, and
+open the :guilabel:`Website payments` menu.
 
-To set the **PDT Identity Token**, switch to :ref:`developer mode <developer-mode>` and retrieve the
-token by following the configuration step :ref:`paypal/enable-pdt`.
+.. image:: paypal/paypal-account.png
+   :align: center
+   :alt: PayPal account menu
 
 .. important::
-   If you are trying Paypal as a test, using a :ref:`Paypal Sandbox account <paypal/testing>`,
-   change the **State** to *Test Mode*. We recommend doing this on a test Odoo database rather than
-   on your main database.
+   Note that for PayPal to work **in Odoo**, the options :ref:`Auto Return <paypal/auto-return>`,
+   :ref:`PDT <paypal/pdt>`, and :ref:`IPN <paypal/ipn>` **must** all be enabled.
```

Comment:
then maybe the pragraph could be re-written as I don't understand why  you added the info that should be performed from the settings it in an important note. We can discuss it if you want :) 
I would remove the important note and explain what should be done from the Paypal settings:
"To access your PayPal account settings, log into PayPal, open the Account Settings, and open the Website payments menu. From here, you must the options XXX must be enabled."

Reply:
Because those options in PayPal are not mandatory for PayPal itself to work, sometimes it's preferences. Customers who used PayPal in other instances or with other software may think some options can be left disabled in Odoo too, which is *not* the case.

Some options *have* to be enabled for the PayPal integration to work with Odoo, even if the customer used to do it differently in its previous software.